src/PABLO/Octant.cpp

```diff
@@ -1584,6 +1584,205 @@ Octant Octant::computePeriodicOctant(uint8_t iface) const {
 		return degOct;
 	}
 
+};
+
+/** Build a same size periodic octant of this octant throught node inode.
+ * \return Periodic octant of the same size (note: it is a stand-alone octant,
+ * may be not living in octree).
+ * The directions through the octant is periodic has to be passed as boolean flags.
+ * \param[in] inode Local index of the node target.
+ */
+Octant Octant::computeNodePeriodicOctant(uint8_t inode) const {
+    Octant degOct(this->m_dim, this->m_level, this->m_x, this->m_y, this->m_z);
+    uint32_t maxLength = uint32_t(1)<<TreeConstants::MAX_LEVEL;
```

Comment:
There is a constant for the max length: MAX_LENGTH in TreeConstants (should be accessible through sm_treeConstants).

Reply:
Ah yes, I think I used some lines above....